### DISCUSSION TITLE
Stylex with storybook and rspack

### DISCUSSION THREAD

**Discussion by luckyluggi** [facebook/stylex#662] [Q&A]
Hello,
I know there has already been a discussion about this bug, but i could not find a solution for it.

I also can not find the issue mentioned here:
https://github.com/facebook/stylex/discussions/292

I want to create a design-system using stylex and react.
I want do use Storybook for documentation, testing and all the cool stuff storybook can do.

I setup a basic project using rspack as a bundler here:
https://github.com/luckyluggi/stylex_storybook

As soon as i start storybook and open a very simple story in the browser the following error does appear:
```
'stylex.create' should never be called at runtime. It should be compiled away by '@stylexjs/babel-plugin'
```

I setup the babel config like described here:
https://stylexjs.com/docs/learn/installation/

I also tried to replicate the example using webpack, wich results in the same error.

Please can anyone help me to solve the issue.
Stylex and Storybook would be the perfect match for me.

**Comment by tounsoo** [comment]
I believe storybook on [dev mode needs to use babel-plugin](https://stylexjs.com/docs/learn/installation/#development). Something similar to this:
```
import styleXPlugin from "@stylexjs/babel-plugin";

const config: StorybookConfig = {
	{...otherConfigStuff}
	async babel(config, { configType }) {
		if (configType === "DEVELOPMENT") {
			return {
				plugins: [
					[
						styleXPlugin,
						{
							dev: true,
							test: false,
							unstable_moduleResolution: {
								type: "commonJS",
								rootDir: __dirname,
							},
						},
					],
				],
			};
		}
		return config;
	},
};
export default config;

```
